pkg/apis/serving/v1alpha2/explainer_alibi.go

```diff
@@ -2,30 +2,25 @@ package v1alpha2
 
 import (
 	"fmt"
-	"strings"
-
 	"github.com/kubeflow/kfserving/pkg/constants"
 	"github.com/kubeflow/kfserving/pkg/utils"
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/core/v1"
+	"strings"
 )
 
 var (
-	AlibiImageName              = "docker.io/seldonio/alibiexplainer"
-	DefaultAlibiRuntimeVersion  = "0.2.3"
-	AllowedAlibiRuntimeVersions = []string{
-		"0.2.3",
-	}
-	InvalidAlibiRuntimeVersionError = "RuntimeVersion must be one of " + strings.Join(AllowedAlibiRuntimeVersions, ", ")
+	AlibiImageName                  = "docker.io/seldonio/alibiexplainer"
```

Comment:
If the configmap defines this, why bother even hardcoding it here?

Reply:
We can validate that the configmap and all these keys exist when starting up the controller, and crash it otherwise.